pxf/pxf-hive/src/main/java/org/apache/hawq/pxf/plugins/hive/HiveColumnarSerdeResolver.java

```diff
@@ -233,4 +238,9 @@ private void resolvePrimitive(Object o, PrimitiveObjectInspector oi) throws IOEx
         }
         firstColumn = false;
     }
+
+    @Override
+    void parseDelimiterChar(InputData input) {
+        delimiter = 44; //,
```

Comment:
Is this a temporary workaround ?
Why not use a static variable to store the delimiter value